package.json

```diff
@@ -144,7 +144,8 @@
     "typescript": "3.5.3"
   },
   "resolutions": {
-    "@types/react": "16.9.1"
+    "@types/react": "16.9.1",
+    "react": "16.9.0"
```

Comment:
Just to experiment. Without continue to end up with two react versions due to `react-intl`.